packages/expo-image/src/Image.tsx

```diff
@@ -42,6 +42,19 @@ export class Image extends React.PureComponent<ImageProps> {
     return await ExpoImageModule.clearDiskCache();
   }
 
+  /**
+   * Asynchronously checks if an image exists in the disk cache and resolves to
+   * the path of the cached image if it does.
+   * @param source - The source URL to retrieve the cached version's path for
+   * @platform android
+   * @platform ios
+   * @return A promise resolving to the path of the cached image. It will resolve
+   * to `false` if the image does not exist in the cache.
+   */
+  static async getCachePathAsync(source: string): Promise<string | false> {
```

Comment:
I would use `null` instead of `false`. `null` generally means a missing value, what does `false` mean in that case?
```suggestion
  static async getCachePathAsync(source: string): Promise<string | null> {
```

Reply:
Hmm, yea. In my head I was thinking `false` as we are saying it doesn't exist....but that's exactly what null would mean huh 😅